File: Sources/Server/fairy-teller/src/main/java/com/rad/fairyteller/controller/UserController.java
package com.rad.fairyteller.controller;

import com.rad.fairyteller.service.UserService;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequiredArgsConstructor
public class UserController {

    private final UserService userService;

    @RequestMapping(path = "/user/delete/{id}")
    public void deleteUser(@PathVariable final Long id) {
        userService.deleteUser(id);
    }
}

File: Sources/Server/fairy-teller/src/main/java/com/rad/fairyteller/controller/UserController.kt
package com.rad.fairyteller.controller

import com.rad.fairyteller.service.UserService
import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class UserController(private val userService: UserService) {

    @RequestMapping(path = arrayOf("/user/delete/{id}"))
    fun deleteUser(@PathVariable id: Long?) {
        userService.deleteUser(id)
    }
}